tools/wptrunner/wptrunner/executors/executormarionette.py

```diff
@@ -306,13 +333,33 @@ def clear_origin(self, url):
         with self.marionette.using_context(self.marionette.CONTEXT_CHROME):
             self.marionette.execute_script(script)
 
+    def find_elements_by_css_selector(self, selector):
+        return self.marionette.find_elements("css selector", selector)
```

Comment:
Instead of these, couldn’t you just called `self.protocol.marionette.find_elements` so you don’t have to reinvent the API?